examples/dave_demos/launch/dave_object.launch.py

```diff
@@ -0,0 +1,165 @@
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, IncludeLaunchDescription, OpaqueFunction
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution, TextSubstitution
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.conditions import IfCondition
+from launch_ros.substitutions import FindPackageShare
+
+
+def launch_setup(context, *args, **kwargs):
+    paused = LaunchConfiguration("paused").perform(context)
+    gui = LaunchConfiguration("gui").perform(context)
+    use_sim_time = LaunchConfiguration("use_sim_time").perform(context)
+    headless = LaunchConfiguration("headless").perform(context)
+    verbose = LaunchConfiguration("verbose").perform(context)
+    namespace = LaunchConfiguration("namespace").perform(context)
+    world_name = LaunchConfiguration("world_name").perform(context)
+    x = LaunchConfiguration("x").perform(context)
+    y = LaunchConfiguration("y").perform(context)
+    z = LaunchConfiguration("z").perform(context)
+    roll = LaunchConfiguration("roll").perform(context)
+    pitch = LaunchConfiguration("pitch").perform(context)
+    yaw = LaunchConfiguration("yaw").perform(context)
+    use_ned_frame = LaunchConfiguration("use_ned_frame").perform(context)
+
+    if world_name != "empty.sdf":
+        world_filename = f"{world_name}.world"
+        world_filepath = PathJoinSubstitution(
+            [FindPackageShare("dave_worlds"), "worlds", world_filename]
+        ).perform(context)
+        gz_args = [world_filepath]
+    else:
+        gz_args = [world_name]
+
+    if headless == "true":
+        gz_args.append("-s")
+    if paused == "false":
+        gz_args.append("-r")
+    if verbose == "true":
+        gz_args.append("--verbose")
+
+    gz_args_str = " ".join(gz_args)
+
+    gz_sim_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("ros_gz_sim"),
+                        "launch",
+                        "gz_sim.launch.py",
+                    ]
+                )
+            ]
+        ),
+        launch_arguments=[
+            ("gz_args", TextSubstitution(text=gz_args_str)),
+        ],
+        condition=IfCondition(gui),
+    )
+
+    object_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("dave_object_models"),
+                        "launch",
+                        "upload_object.launch.py",
+                    ]
+                )
+            ]
+        ),
+        launch_arguments={
+            "gui": gui,
+            "use_sim_time": use_sim_time,
+            "namespace": namespace,
+            "x": x,
+            "y": y,
+            "z": z,
+            "roll": roll,
```

Comment:
I've just found out that the launch argument `roll`, `pitch` ,`yaw` does not work. It has to be `R`, `P`, `Y`. We should modify from line 80-82, 20-22, and any other launch.py that uses this.

Reply:
@woensug-choi I just tried launching with all the arguements and seems to work fine. I used the following command:

`ros2 launch dave_demos dave_sensor.launch.py namespace:=nortek_dvl500_300 world_name:=dvl_world paused:=false x:=0 y:=1.2 z:=-20 roll:=1.57 pitch:=0.0 yaw:=0.0 headless:=false gui:=true verbose:=true paused:=false`

`roll`, `pitch`, and `yaw` are in radians.

I'm not sure why it does not work for you.

Reply:
If it worked for you. never mind for this PR (as it is not critically related to this)

Reply:
Actually, you are right. It should be R, R, Y.

I'll make the changes!

Reply:
Strange! `ros2 launch dave_demos dave_sensor.launch.py namespace:=nortek_dvl500_300 world_name:=dvl_world paused:=false x:=0 y:=1.2 z:=-20 roll:=1.57 pitch:=0.0 yaw:=0.0 headless:=false gui:=true verbose:=true paused:=false` works find on ubuntu native install machine but causes context error in Mac. (again, if it's only mac problem. ignore for sake of this PR).

Reply:
Strange! It works fine for me both on ubuntu native and mac.

@woensug-choi do you think the problem could be coming from [here](https://github.com/IOES-Lab/dave/blob/1c073e15138baa0f68d0034a4343a97e2cd43b7d/gazebo/dave_ros_gz_plugins/src/DVLBridge.cc#L82)?

I have committed some changes. Please check if the error still occurs.

Reply:
@rakeshv24 Yes it seems... but don't know why...!

Reply:
@woensug-choi it seems that the problem can be resolved by properly shutting down the node. But since this actually does not hinder with the functionality of the code, we can maybe resolve it later. What do you think?